test/units/TEST-17-UDEV.verify.sh

```diff
@@ -292,10 +292,7 @@ test_syntax_error 'OWNER-="b"' 'Invalid operator for OWNER.'
 test_syntax_error 'OWNER!="b"' 'Invalid operator for OWNER.'
 test_syntax_error 'OWNER+="0"' "OWNER key takes '=' or ':=' operator, assuming '='."
 # numeric system UID is valid even if it does not exist
-SYS_UID_MAX=999
-if [[ -e /etc/login.defs ]]; then
-    SYS_UID_MAX=$(awk '$1 == "SYS_UID_MAX" { print $2 }' /etc/login.defs)
-fi
+SYS_UID_MAX="$(userdbctl user -S --no-legend --no-pager | grep 'end system' | awk '{print $8}')"
```

Comment:
userdbd is an optional component, so please add at the top of the test script something like:

```
if ! command userdbctl >/dev/null; then
    echo "command userdbctl not found, skipping..." | tee --append /skipped
    exit 77
fi
```